### PR TITLE
Dynamically retrieve sender using session hook, correctly display on Outlook.

### PR DESCRIPTION
**Pull request by surbhi-gulati**
Fixes [#46](https://github.com/HackBeanpot/internal-tools/issues/46).

- Removes hardcoding of email sender by extracting name and email of sender from useSession hook to form Mailgun request
- Displays sender in pretty format on Outlook by using "h:sender" field to override default mailgun behavior of saying Mailgun sent email on behalf of HBP domain

 (ie sender reads "Surbhi Gulati <surbhi@hackbeanpot.com>" rather than "surbhi=hackbeanpot.com@mailgun.hackbeanpot.com on behalf of Surbhi Gulati <surbhi@hackbeanpot.com>")

After fixing both of these, email renders like so:
7_16 outlook email

### CODE REVIEW

**Review by jzhzhang66 — APPROVED** [review]
LGTM!! There was an issue on main so after you merged main into your branch the email sender page stopped compiling.

I just pushed a fix for the issue on main so please pull from main and then merge main into your branch and make sure everything is working then you're good to merge after dhivas' review / approval! 

@Dhivas-Sugumar to test this branch (if it's still not compiling when u review) you can just remove the StyledFinalMessageContent for now
Screen Shot 2022-07-17 at 10 48 56 AM

File: pages/emailSender.tsx
Comment (jzhzhang66) [inline pages/emailSender.tsx:67]: jk I just realized status is never used - let's remove it

**Review by jzhzhang66 — CHANGES_REQUESTED** [review]

**Review by Dhivas-Sugumar — APPROVED** [review]
LGTM! Great PR!

**Review by jzhzhang66 — APPROVED** [review]

**Pull request merged**
